mpqp/qasm/open_qasm_2_and_3.py

```diff
@@ -46,21 +47,22 @@ class Instr(Enum):
     """Special instruction for which the definition needs to included in the
     file."""
 
-    STD_LIB = 0
-    CSX = 1
-    U0 = 2
-    CU3 = 3
-    SXDG = 4
-    RZZ = 5
-    RXX = 6
-    RCCX = 7
-    RC3X = 8
-    C3X = 9
-    C4X = 10
-    C3SQRTX = 11
-    OQASM2_ALL_STDGATES = 12
-    OQASM3_ALL_STDGATES = 13
-    BRAKET_CUSTOM_INCLUDE = 14
+    STD_LIB = auto()
+    QE_LIB = auto()
+    CSX = auto()
+    U0 = auto()
+    CU3 = auto()
+    SXDG = auto()
+    RZZ = auto()
+    RXX = auto()
+    RCCX = auto()
+    RC3X = auto()
+    C3X = auto()
+    C4X = auto()
+    C3SQRTX = auto()
+    OQASM2_ALL_STDGATES = auto()
+    OQASM3_ALL_STDGATES = auto()
+    BRAKET_CUSTOM_INCLUDE = auto()
```

Comment:
AWS Braket has his own "funky" way of writing OpenQASM3 (`cx` is written `cnot` for instance, `phase` is written `phaseshift`, `ccx` is `ccnot`, `gphase` is not known, and many others). 

In the context of receiving an OpenQASM3 code from Braket circuit, we will have to handle that. We will have to detect when there are some keywords specific to braket and replace them by the right "original" OpenQASM3 one. Or we can also define a file that redefines all those keywords (`cnot`, ...) using native gates, like it is done in `braket_custom_include.inc`, but in the opposite way.